pkg/test/verify/cluster.go

```diff
@@ -43,19 +43,22 @@ var testCases = map[string]struct {
 }
 
 func AddToolchainClusterAsMember(t *testing.T, functionToVerify FunctionToVerify) {
+	t.Helper()
 	// given
 	defer gock.Off()
 	status := test.NewClusterStatus(toolchainv1alpha1.ConditionReady, corev1.ConditionTrue)
 
 	t.Run("add member ToolchainCluster with namespace label set", func(t *testing.T) {
 		for testName, data := range testCases {
 			t.Run(testName, func(t *testing.T) {
-
-				toolchainCluster, sec := test.NewToolchainCluster("east", test.HostOperatorNs, "secret", status, Labels("member-ns", test.NameHost))
+				toolchainCluster, sec := test.NewToolchainCluster(t, "east", test.HostOperatorNs, "member-ns", "secret", status, data.insecure)
```

Comment:
TBH I was a bit confused by the usage of the new `data.insecure` parameter. It seems to drive the `InsecureSkipTLSVerify` flag in the kubeconfig, even if it's not explicitly/directly being checked in the tests. 

It also doesn't drive the configuration of the `DisabledTLSValidations` parameter in the `toolchaincluster` resource.

Reply:
As far as I am aware `DisabledTLSValidations` is not used and is going away in favor of the configuration provided in the kubeconfig.

I will take a closer look at the testcases though.. I was just desperately trying to make them pass without paying too much attention to whether they actually make sense in the shape they are now.

Reply:
that being said, you can drop https://github.com/codeready-toolchain/toolchain-common/blob/a718700447063d91735563f4c2d2b83ee5615e05/pkg/test/verify/cluster.go#L25 as well